### PR TITLE
VPA - Moving status updater to utils

### PR DESCRIPTION
**Pull request by krzysied**
Moving status_updater to utils/.
This is a setup for e2e tests.

ref #2738

### CODE REVIEW

File: vertical-pod-autoscaler/pkg/utils/status/status_updater.go [hunk 1]
```diff
@@ -14,48 +14,46 @@ See the License for the specific language governing permissions and
...
 )
 
-const (
-	updateInterval = 10 * time.Second
-)
-
-// StatusUpdater periodically updates Admission Controller status.
-type StatusUpdater struct {
-	client *status.Client
+// Updater periodically updates Admission Controller status.
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/utils/status/status_updater.go]: Remove Admission Controller from comment

File: vertical-pod-autoscaler/pkg/utils/status/status_updater.go [hunk 2]
```diff
@@ -14,48 +14,46 @@ See the License for the specific language governing permissions and
...
+type Updater struct {
```
Comment (bskiba) [inline vertical-pod-autoscaler/pkg/utils/status/status_updater.go]: Soooo we already have an updater in VPA :)
Can you think of a similar name that will not cause clashes?

File: vertical-pod-autoscaler/pkg/utils/status/status_updater.go (hunk 1 above)
Comment (krzysied) [reply vertical-pod-autoscaler/pkg/utils/status/status_updater.go]: Done.

File: vertical-pod-autoscaler/pkg/utils/status/status_updater.go (hunk 2 above)
Comment (krzysied) [reply vertical-pod-autoscaler/pkg/utils/status/status_updater.go]: In normal cases the Updater will be referenced by the package name. Creating it looks like this:
```
status.NewUpdater(...)
```
So there should be no confusion there.

**Comment by krzysied** [comment]
@bskiba PTAL

**Pull request merged**